koach/common/common.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// ================ //
+// == Kubernetes == //
+// ================ //
+
+// IsK8sLocal Function
+func IsK8sLocal() bool {
```

Comment:
Ig these functions are same as KubeArmor, Can we import it from there?

Reply:
sure

Reply:
I decide to not import from KubeArmor because it uses runtime configuration on determining if it is k8slocal or not https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/common/common.go#L322

Reply:
Makes sense.